.github/workflows/build.yml

```diff
@@ -54,9 +54,9 @@ jobs:
       
       - name: Trigger QA Deployment
         if: ${{ success() }}
-        run: |
-          curl -X POST \
-               -H "Accept: application/vnd.github.v3+json" \
-               -H "Authorization: Bearer ${{ secrets.ACTIONS_API_ACCESS_TOKEN }}" \
-               https://api.github.com/repos/DFE-Digital/find-teacher-training/deployments \
-               -d '{"ref":"${{ github.sha }}", "environment": "qa", "required_contexts": []}'
+        uses: benc-uk/workflow-dispatch@v1.1
+        with:
+          workflow: 3074188 # deploy.yml from https://api.github.com/repos/DFE-Digital/find-teacher-training/actions/workflows
```

Comment:
Can we use the workflow name instead? As per https://github.com/benc-uk/workflow-dispatch#workflow

Reply:
done, but we have to remember to update references if the name changes.